dotcom-rendering/src/components/LatestLinks.importable.tsx

```diff
@@ -121,21 +115,28 @@ export const LatestLinks = ({
 			: palette.neutral[86]};
 	`;
 
+	const height = isDynamo
+		? `calc(5px + 4 * ${lineHeights.regular}em)`
+		: `calc(4 * ${lineHeights.regular}em)`;
+
+	const ulStyle = css`
+		display: flex;
+		gap: 5px;
+		padding-bottom: ${space[2]}px;
+		box-sizing: border-box;
+		min-height: ${height};
```

Comment:
We should probably set the `height` on the parent or the child exclusively.

I think it’s best to use either use `min-height`/`max-height` or an explicit `height`. Otherwise, the CSS is at best redundant and at worse impossible to satisfy.

Here, I think we should just set the `height` explicitely, once, on the parent, to 4 &times; the `line-height` plus padding if Dynamo, as you’ve done:

```ts
/* not min-height, just good ol’ height */
const height = isDynamo
		? `calc(5px + 4 * ${lineHeights.regular}em)`
		: `calc(4 * ${lineHeights.regular}em)`;
```